ui/apps/dashboard/src/components/DatadogIntegration/SetupPage.tsx

```diff
@@ -283,7 +293,10 @@ export default function SetupPage({
                         <span className="italic">Setting up…</span>
                       </div>
                     ) : (
-                      <></>
+                      <div className="font-normal">
+                        <span className="text-error font-bold">Error:</span>{' '}
+                        <code>Please contact Inngest Support.</code>
```

Comment:
This last case is a "should never happen" case but is here just to exhaustively cover all bases.